routes/web.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
-use App\Http\Controllers\AirlineController;
-use App\Http\Controllers\CityController;
+use App\Http\Controllers\AirlineViewController;
+use App\Http\Controllers\CityViewController;
```

Comment:
```suggestion
use App\Http\Controllers\AirlineController;
use App\Http\Controllers\CityController;
```
The `View` keyword isn't really necessary. If a project structure has an `Api` directory within `Controllers` we implicitly know that all controllers outside of that directory are meant for functionalities not API-specific.